### ISSUE TITLE
bluem-php depends on selective/xmldsig which is abandonded.

### ISSUE THREAD

**Issue by bramlinssen** [bluem-development/bluem-php#90]
This library depends on selective/xmldsig which is abandonded.
Installing bluem-php gives the notification 
`Package selective/xmldsig is abandoned, you should avoid using it. Use robrichards/xmlseclibs instead.`

**Comment by DaanRijpkema** [comment]
Yes, thanks for mentioning this @bramlinssen . I'm working on adding a newer dependency here: https://github.com/bluem-development/bluem-php/pull/83

**Comment by bramlinssen** [comment]
Hi @DaanRijpkema  This issue still hasn't been solved. When can we accept this pull request to be merged in the main branch?
